google/cloud/internal/attributes.h

```diff
@@ -36,14 +33,9 @@
 #error "Applications should not define GOOGLE_CLOUD_CPP_DEPRECATED"
 #elif GOOGLE_CLOUD_CPP_DISABLE_DEPRECATION_WARNINGS
 // Do nothing, the default definition effectively
-#elif GOOGLE_CLOUD_CPP_CPP_VERSION >= 201402L
+#else
 #define GOOGLE_CLOUD_CPP_DEPRECATED(message) [[deprecated(message)]]
-#elif defined(_MSC_VER)
-#define GOOGLE_CLOUD_CPP_DEPRECATED(message) __declspec(deprecated(message))
-#elif defined(__clang__) || defined(__GNUC__)
-#define GOOGLE_CLOUD_CPP_DEPRECATED(message) \
-  __attribute__((deprecated(message)))
-#endif  // C++ <= 11
+#endif
 
 #ifndef GOOGLE_CLOUD_CPP_DEPRECATED
```

Comment:
Now that there is an unconditional `#else` branch, we might as well move the do-nothing `GOOGLE_CLOUD_CPP_DEPRECATED()` definition to the "Do nothing" branch.

Reply:
Good point. Fixed.